server/rpc.jai

```diff
@@ -29,7 +30,10 @@ read_from_stdin :: (buffer: *u8, bytes_to_read: u32) -> u32 {
               readed += xx read_this_time;
```

Comment:
Note that, on POSIX this return `s64` which you add to `u32` which might cause problems if it actually manages to read more bytes that it can handle. @SogoCZE

Reply:
This should be fixed in https://github.com/SogoCZE/Jails/commit/76367c2bef3e5e8fa9e3f1ef5926ee658e23b8cf. Thanks!